modules/core/src/v2/coins/eos.ts

```diff
@@ -319,7 +321,7 @@ export class Eos extends BaseCoin {
     if (destinationDetails.pathname === address) {
       return {
         address: address,
-        memoId: undefined,
+        memoId: '',
```

Comment:
On the platform side, we set it to `null`.

https://github.com/BitGo/bitgo-microservices/blob/develop/packages/wallet-platform/app/controllers/api/v2/coins/eos.js#L360

Should it be same here in sdk as well?

Reply:
this might be an issue - wasn't aware that this code was duplicated here as well. will need to update this to match the SDK.

the problem with `verifyTransaction` is that when we deserialize a transaction without a memoId, it comes back as empty string instead of undefined.